R/get_-functions.R

```diff
@@ -262,7 +277,7 @@ get_protected_columns <- function(data = NULL) {
 
   protected_columns <- c(
```

Comment:
this is another pain point. I guess we could have a s3 method for all classes that lists the protected columns for that ? Are we using this internally anywhere we don't know about the class of the input the user has made? If no we could do this and make this a lot more modular/hard coded

Reply:
Issue: https://github.com/epiforecasts/scoringutils/issues/889